app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    bed_result = []
```

Comment:
The variable name `bed_result` seems to be a typo. It should likely be `bad_result` to reflect its purpose of storing negative outcomes.